beezee/assetlist.json

```diff
@@ -20,8 +20,14 @@
       "name": "BeeZee",
       "display": "bze",
       "symbol": "BZE",
+      "ibc": {
+        "source_channel": "channel-0",
+        "dst_channel": "channel-340",
```

Comment:
which chain uses channel 340? This info should be defined in the \_IBC\ folder, not here